pkg/cloudprovider/config.go

```diff
@@ -27,7 +27,7 @@ import (
 func ValidateObjectStoreConfigKeys(config map[string]string, validKeys ...string) error {
 	// `bucket` is automatically added to all object store config by
```

Comment:
pls update this comment, and the godoc comment on this function, to reference "prefix" as well.